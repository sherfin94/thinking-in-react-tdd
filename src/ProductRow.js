import React, {Component} from 'react'

function ProductRow(props) {
  return(
    <tr>
      <td> {props.name} </td>
      <td> {props.price} </td>
    </tr>
  )
}

module.exports = ProductRow
