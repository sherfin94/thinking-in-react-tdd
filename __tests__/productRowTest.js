import React from 'react';
import ProductRow from '../src/ProductRow';
import renderer from 'react-test-renderer';

describe("Product row", function() {

  it(' matches snapshot ', () => {
    const component = renderer.create(
      <ProductRow name = "FootBall" price = "50" />
    );
    let json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
