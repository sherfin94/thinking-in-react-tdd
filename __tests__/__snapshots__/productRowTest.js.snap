exports[`Product row  matches snapshot  1`] = `
<tr>
  <td>
     
    FootBall
     
  </td>
  <td>
     
    50
     
  </td>
</tr>
`;
